website/pages/resources/index.jsx

```diff
@@ -58,371 +56,311 @@ export default function ResourcesPage() {
-                <ul>
-                  <li>
-                    <a href="https://www.hashicorp.com/resources/going-cloud-native-at-the-dutch-national-police">
-                      Going Cloud-Native at the Dutch National Police
-                    </a>
-                  </li>
-                </ul>
-                <li>N26</li>
-                <ul>
-                  <li>
-                    <a href="https://medium.com/insiden26/tech-at-n26-the-bank-in-the-cloud-e5ff818b528b">
-                      Tech at N26 - The Bank in the Cloud
-                    </a>
-                  </li>
-                </ul>
-                <li>Elsevier</li>
-                <ul>
-                  <li>
-                    <a href="https://www.hashicorp.com/resources/elsevier-nomad-container-framework-demo">
-                      Eslevier&apos;s Container Framework with Nomad, Terraform,
-                      and Consul
-                    </a>
-                  </li>
-                </ul>
-                <li>Graymeta</li>
-                <ul>
-                  <li>
-                    <a href="https://www.hashicorp.com/resources/backend-batch-processing-nomad">
-                      Backend Batch Processing At Scale with Nomad
-                    </a>
-                  </li>
-                </ul>
-                <li>NIH NCBI</li>
-                <ul>
-                  <li>
-                    <a href="https://www.hashicorp.com/resources/ncbi-legacy-migration-hybrid-cloud-consul-nomad">
-                      NCBI’s Legacy Migration to Hybrid Cloud with Consul &amp;
-                      Nomad
-                    </a>
-                  </li>
-                </ul>
-                <li>Q2EBanking</li>
-                <ul>
-                  <li>
-                    <a href="https://www.youtube.com/watch?v=OsZeKTP2u98&feature=youtu.be&t=1499">
-                      Q2&apos;s Nomad Use and Overview
-                    </a>
-                  </li>
-                </ul>
-                <li>imgix</li>
+              </p>
+              <p>
+                <strong>Bug Tracker</strong>
                 <ul>
                   <li>
-                    <a href="https://medium.com/@copyconstruct/schedulers-kubernetes-and-nomad-b0f2e14a896">
-                      Cluster Schedulers &amp; Why We Chose Nomad Over
-                      Kubernetes
+                    <a href="https://github.com/hashicorp/nomad/issues">
+                      GitHub Issues
                     </a>
                   </li>
                 </ul>
-                <li>Region Syddanmark</li>
+                Please only use this to report bugs. For general help, please
+                use our mailing list or Gitter.
+              </p>
+              <h3>Who Uses Nomad</h3>
+              <h4>Roblox</h4>
+              <ul>
+                <li>
+                  <a href="https://www.hashicorp.com/case-studies/roblox/">
```

Comment:
it seems a little weird to have these next to each other... is there something interesting in the 70M talk that adds to the 100M talk, or are we trying to highlight growth/continuity?

Reply:
The two talks are different and when combined show the technical evolution over a single year they've done to keep up with their scale. 🥇